src/main/resources/data/pieces/268758-03_02_create_by_none.json

```diff
@@ -3,7 +3,6 @@
   "caption": "Tutorial Volume 6",
   "comment": "Limited Edition",
   "format": "Electronic",
-  "itemId": "15447c41-bc6a-4600-96a4-a1ce7f44c62a",
   "locationId": "fcd64ce1-6995-48f0-840e-89ffa2288371",
```

Comment:
Either there shouldn't be "locationId" or "locationId" should be https://github.com/folio-org/mod-orders-storage/blob/974687774bb139f8f0651b7f33d7533721d0abaf/src/main/resources/data/po-lines/268758-03_fully_received_electronic_resource.json#L67